generate-release/src/main.rs

```diff
@@ -15,33 +17,26 @@ mod release_notes_website;
 
 /// Generates markdown files used for a bevy releases.
 ///
-/// Migration Guide:
-/// * Gets all merged PRs with the `C-Breaking-Change` label.
-/// * For each PR:
-///     * Generate the title with a link to the relevant PR and
-///     * Generate the migration guide section. This parses the markdown and generates valid makrdown that should pass markdownlint rules.
-///
-/// Release notes:
-/// * Gets all merged PRs
-/// * Collect each author of closed PRs (Should this just list all contributors?)
-/// * Sort each PR per area label
-/// * Generate the list of merge PR
-///
 /// Requires a valid `GITHUB_TOKEN` environment variable, you can use a .env file or use your preferred method of passing env arguments.
 ///
-/// Example used to generate for 0.9:
-/// cargo run -- migration-guide --from v0.9.0 --to main --title "0.9 to 0.10" --weight 6
-/// cargo run -- release-note --from v0.9.0 --to main
+/// Example used to generate the 0.14 release:
+/// cargo run -- migration-guide --from v0.13.0 --to main --release-version 0.14
+/// cargo run -- combine-migration-guides --title "0.13 to 0.14" --weight 9 --release-version 0.14
```

Comment:
Are we able to automate generation of the "weight" parameter?

Reply:
Not easily, that would require parsing all previous guides to figure out the latest and also updating introduction.md

It's technically possible, but way out of scope

Reply:
Yep, definitely for follow-up!